src/webgl/p5.Geometry.js

```diff
@@ -1910,6 +1919,119 @@ p5.Geometry = class Geometry {
+ * attributes can also be defined within `beginShape()` and `endShape()`.
+ * 
+ * The first parameter, `attributeName`, is a string with the attribute's name.
+ * This is the same variable name which should be declared in the shader, similar to
+ * `setUniform()`.
+ * 
+ * The second parameter, `data`, is the value assigned to the attribute. This value
+ * will be pushed directly onto the Geometry object. There should be the same number 
+ * of custom attribute values as vertices.
+ * 
+ * The `data` can be a Number or an array of numbers. Tn the shader program the type
+ * can be declared according to the WebGL specification. Common types include `float`, 
+ * `vec2`, `vec3`, `vec4` or matrices.
+ * 
+ * See also the global <a href="#/p5/setAttribute">setAttribute()</a> function.
+ * 
+ * @example
+ * <div>
+ * <code>
+ * let geo;
+ * 
+ * function cartesianToSpherical(x, y, z) {
+ *   let r = sqrt(pow(x, x) + pow(y, y) + pow(z, z));
+ *   let theta = acos(z / r);
+ *   let phi = atan2(y, x);
+ *   return { theta, phi };
+ * }
+ * 
+ * function setup() {
+ *   createCanvas(100, 100, WEBGL);
+ * 
+ *   // Modify the material shader to display roughness.
+ *   const myShader = materialShader().modify({
+ *     vertexDeclarations:`in float aRoughness;
+ *                         out float vRoughness;`,
+ *     fragmentDeclarations: 'in float vRoughness;',
+ *     'void afterVertex': `() {
+ *         vRoughness = aRoughness;
+ *     }`,
+ *     'vec4 combineColors': `(ColorComponents components) {
+ *             vec4 color = vec4(0.);
+ *             color.rgb += components.diffuse * components.baseColor * (1.0-vRoughness);
+ *             color.rgb += components.ambient * components.ambientColor;
+ *             color.rgb += components.specular * components.specularColor * (1.0-vRoughness);
+ *             color.a = components.opacity;
+ *             return color;
+ *     }`
+ *   });
+ * 
+ *   // Create the Geometry object.
+ *   beginGeometry();
+ *   fill('hotpink');
+ *   sphere(45, 50, 50);
+ *   geo = endGeometry();
+ * 
+ *   // Set the roughness value for every vertex.
+ *   for (let v of geo.vertices){
+ *     // convert coordinates to spherical coordinates
+ *     let spherical = cartesianToSpherical(v.x, v.y, v.z);
+ *     let roughness = noise(spherical.theta*5, spherical.phi*5);
+ *     geo.setAttribute('aRoughness', roughness);
+ *   }
+ * 
+ *   // Use the custom shader.
+ *   shader(myShader);
+ *   describe('A rough pink sphere rotating on a blue background.');
+ * }
+ * 
+ * function draw() {
+ *   // Set some styles and lighting
+ *   background('lightblue');
+ *   noStroke();
+ * 
+ *   specularMaterial(255,125,100);
+ *   shininess(2);
+ *
+ *   directionalLight('white', -1, 1, -1);
+ *   ambientLight(320);
+ *
+ *   rotateY(millis()*0.001);
+ *
+ *   // Draw the geometry
+ *   model(geo);
+ * }
+ * </code>
+ * </div>
+/
+/**
+ * @method setAttribute
```

Comment:
Same thing here for making it one doc comment, and also I think for methods on a class (as opposed to things we define via `p5.prototype.methodname = ...`), we don't need the `@method` tag any more.

Reply:
Fixed these now :)